cli/lsp/code_lens.rs

```diff
@@ -508,6 +513,14 @@ mod tests {
       Deno.test.only("test only", () => {});
 
       Deno.test(`test template literal name`, () => {});
+
+      describe("test describe", () => {});
+
+      describe.ignore("test describe ignore", () => {});
+
+      describe.only("test describe only", () => {});
+
+      describe.skip("test describe skip", () => {});
```

Comment:
Woah, I didn't know this was supported

Reply:
Ignore, skip, and only were supported since the beginning. It makes it easy to skip or focus test cases or groups during development. I think ignore and skip are the same, one being an alias for the other.